charts/monitoring-config/rules/email_alert_api_alerts_tests.yaml

```
@@ -0,0 +1,96 @@
+rule_files:
+  - email_alert_api_alerts.yaml
+
+evaluation_interval: 1m
+
+tests:
+  # Alert does not fire if current and delivered are identical
+  - name: travel_advice_alert_test_identical
+    interval: 1m
+    input_series:
+      - series: 'email_alert_api_current_travel_advice_alerts'
+        values: '2'
+      - series: 'email_alert_api_delivered_travel_advice_alerts'
+        values: '2'
+
+    alert_rule_test:
+      - alertname: TravelAdviceAlert
+        exp_alerts: []
+
+  # Alert does not fire if current and delivered are zero
+  - name: travel_advice_alert_test_zeros
+    interval: 1m
+    input_series:
+      - series: 'email_alert_api_current_travel_advice_alerts'
+        values: '0'
+      - series: 'email_alert_api_delivered_travel_advice_alerts'
+        values: '0'
+
+    alert_rule_test:
+      - alertname: TravelAdviceAlert
+        exp_alerts: []
+
+  # Alert fires if current and delivered are different
+  - name: travel_advice_alert_test
+    interval: 1m
+    input_series:
+      - series: 'email_alert_api_current_travel_advice_alerts'
+        values: '1'
+      - series: 'email_alert_api_delivered_travel_advice_alerts'
+        values: '0'
+
+    alert_rule_test:
+      - alertname: TravelAdviceAlert
+        exp_alerts:
+          - exp_labels:
+              severity: page
+            exp_annotations:
+              summary: No delivered emails for Travel Advice update created >1 hour ago
+              # yamllint disable-line rule:line-length
+              runbook_url: 'https://docs.publishing.service.gov.uk/manual/alerts/email-alerts-travel-medical.html'
```

The linter wants you to do this:
```suggestion
              runbook_url:
                https://docs.publishing.service.gov.uk/manual/alerts/email-alerts-travel-medical.html
```
(it should just fix stuff like this automatically rather than complaining ofc, but GitHub makes that awkward to implement)